### PR TITLE
fix: terraform fmt --recursive

### LINKED ISSUE #210: terraform fmt --recursive
The built in "terraform fmt" command is a non breaking formatting tool to adhere to the terraform style guide.
Ideally all pull requests should adhere to the terraform style guide. Maybe in the future we can add an automated workflow to run "terraform fmt" on all pull requests before merge.

### PR DESCRIPTION
**Pull request by robinmordasiewicz**
- fix #210 

The terraform fmt command is used to rewrite Terraform configuration files to a canonical format and style. This command applies a subset of the [Terraform language style conventions](https://developer.hashicorp.com/terraform/language/syntax/style), along with other minor adjustments for readability.

This change is non breaking and is to help follow coding standards in the terraform style guide.

### CODE REVIEW

File: ibm/7.0/ha/fgt.tf [hunk 1]
```diff
@@ -106,15 +106,15 @@ data "template_file" "userdata" {
...
-    license_file = "${file("${var.license}")}"
-    hapriority   = "255"
-    apikey       = "${var.ibmcloud_api_key}"
-    port1_ip     = "${var.fgtaport1}"
-    port1_mask   = "${var.fgtaport1mask}"
-    port2_ip     = "${var.fgtaport2}"
-    port2_mask   = "${var.fgtaport2mask}"
-    port3_ip     = "${var.fgtaport3}"
-    port3_mask   = "${var.fgtaport3mask}"
+    license_file    = "${file("${var.license}")}"
```
Comment (Joel-Cripps) [inline ibm/7.0/ha/fgt.tf:109]: surprised these weren't caught in the fmt.

**Review by Joel-Cripps — CHANGES_REQUESTED** [review]
If we can change those few vars that still have the quotes around them that would be ideal.

File: ibm/7.0/ha/fgt.tf (hunk 1 above)
Comment (robinmordasiewicz) [reply ibm/7.0/ha/fgt.tf:109]: @Joel-Cripps I'm not sure what the change request is ?

Comment (robinmordasiewicz) [reply ibm/7.0/ha/fgt.tf:109]: @Joel-Cripps I think I understand what you meant. I do see the fmt command didn't adjust the values inside the the data resource, and I see there is some inconsistencies. I will review to see if there are other areas that need attention, and I will update the PR

**Pull request merged**